### PR TITLE
WIP Update and Switch over to github actions

### PR DESCRIPTION
**Pull request by carkod**
## Done

Replace Circle CI with Github actions

## QA

Checks should turn green

## Issue / Card

Testing sprint

Blocked by https://github.com/canonical-web-and-design/dotrun/issues/33

### CODE REVIEW

**Comment by carkod** [comment]
@nottrobin getting permission denied to dotrun, not sure why...

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        run: |
+          sudo snap install dotrun
+          sudo chown root:root /
+
+      - name: Install dependencies
+        run: dotrun install
+
+      - name: Build assets
+        run: |
+          sudo chown root:root /etc/gitconfig
```
Comment (nottrobin) [inline .github/workflows/pr.yaml:36]: Try changing this to
```suggestion
          sudo rm /etc/gitconfig
```

**Comment by jpmartinspt** [comment]
@carkod revisit see if this is still blocked

**Comment by nottrobin** [comment]
I suspect this is never going to happen.

**Pull request closed without merging**